src/common/Card.tsx

```diff
@@ -0,0 +1,64 @@
+import { FC, ReactNode, useState } from "react";
+import styled, { css } from "styled-components";
+import { BiUpArrow, BiDownArrow } from "react-icons/bi";
+
+const cardCss = css`
+  background-color: #4a5568;
+  color: #f5f5f5;
+  box-shadow: 0 1px 2px rgb(33 34 35/50%), 0 1px 2px rgb(26 26 27/50%);
+  border-radius: 0.5rem;
+  padding: 1rem;
+  margin: 1.25rem;
+  height: min-content;
+
+  & > h2 {
+    margin: 0;
+  }
+`;
+
+export const Card = styled.article`
+  ${cardCss}
+`;
```

Comment:
What do you think about having the cardCss inside Card, and letting DropdownCardContainer extend Card, like `DropdownCardContainer = styled(Card)`?

Reply:
Very nice! Wasn't really sure how to extend, but it worked neatly.